File: labflutter/lib/product.dart
import 'package:uuid/uuid.dart';

class Product {
  final String id = const Uuid().v4();
  String name;
  String type;
  String description;
  String quantity;
  String price;

  Product({
    required this.name,
    required this.type,
    required this.description,
    required this.quantity,
    required this.price
  });
}

File: LabServer/Lab1NativeUI/app/src/main/java/com/example/lab1nativeui/model/Product.kt
package com.example.lab1nativeui.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import java.util.*

@Parcelize
data class Product(
    val id: Int,
    val name: String,
    val type: String,
    val description: String,
    val quantity: Int,
    val price: Double
): Parcelable
